dependencies.yaml

```diff
@@ -146,6 +176,20 @@ dependencies:
           - matrix: # All CUDA 11 versions
             packages:
               - &cuda_python11 cuda-python>=11.7.1,<12.0a0
+      - output_types: [requirements, pyproject]
```

Comment:
Are we able to write matrices to `pyproject` now or are devcontainers building wheels (or rather, building a python package) by writing matrices to `requirements.txt`?

Reply:
Once https://github.com/rapidsai/dependency-file-generator/pull/48 is merged yes, generating pyproject.toml via matrices will work. But the devcontainers currently build a combined `requirements.txt` for every project in the container, and that already works with matrices.